soccer/gameplay/main.py

```diff
@@ -201,6 +204,7 @@ def run():
     try:
         if root_play() is not None:
             root_play().spin()
+            situationAnalysis.updateAnalysis()
```

Comment:
It's a 1 frame difference in lag, but is there any reason not to have this run before the root play?